modules/splunk-configuring-acs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_module-type: PROCEDURE
+[id="splunk-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the token value.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (*`add`* icon).
```

Comment:
I don't think the other places where you put (add icon) etc. in parentheses used the backticks around it. Same for others below

Reply:
I have had a discussion with Bob and we agreed to use this format. I've changed it for other PR's as well, I'll updated all other to match this style if I messed something.

Reply:
Cool sounds good!